### PR TITLE
It just wanted to be named smqueue-public, thats all.

### PR DESCRIPTION
**Pull request by kkroo**
Ok this one works as `smqueue-public` back to `sipauthserve` to find wtf is going on

### CODE REVIEW

**Comment by kkroo** [comment]
PTAL

File: debian/control [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-Maintainer: Range Networks, Inc. <info@rangenetworks.com>
-Homepage: http://www.rangenetworks.com/
+Maintainer: Endaga, Inc. <hello@endaga.com>
+Homepage: http://www.endaga.com/
 Build-Depends: build-essential, debhelper (>= 7), libtool, pkg-config, autoconf, libsqlite3-dev, libosip2-dev, libzmq3-dev, libcoredumper1, libcoredumper-dev
 Standards-Version: 3.7.3
 
 Package: smqueue-public
-Provides: smqueue
+Provides: smqueue-public
```
Comment (shaddi) [inline debian/control]: Provides should be just `smqueue`

**Comment by shaddi** [comment]
This LGTM (compiles, builds, installs, runs fine); however, we need #6 merged in to fix a dependency issue.

**Comment by shaddi** [comment]
Can we get this merged in so I can start testing it? (er, where it = installing it alongside other packages)

**Comment by kkroo** [comment]
Merged. Happy testing :)

**Pull request merged**